tslint-packages.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tslint.json",
   "linterOptions": {
-    "exclude": ["**/__tests__/**/*"]
+    "exclude": ["**/__tests__/**/*", "packages/rest/index.ts"]
```

Comment:
😕 Could we try to resolve it some other way?

Reply:
potentially the `export * from` will fix this.